lblod/frontend-gelinkt-notuleren
# GN-4200: Implement new TOC scrolling functionality

### Overview
Implement configuration for TOC scrolling to put the element on top of the page

##### connected issues and PRs:
[jira ticket](https://binnenland.atlassian.net/browse/GN-4200)
[Plugins PR](https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/177)


### Setup
Needs to npm link the plugins repo, I will update the PR when the change is released

### How to test/reproduce
1 - Npm the plugins repo
2 - Go to regulatory statements
3 - create a document with a TOC
4 - click on a link of the TOC
5 - The element clicked should be at the top of the page

### Challenges/uncertainties
none


### Checks PR readiness
